.travis.yml

```
@@ -25,7 +25,7 @@ matrix:
       script:
       - if [[ "$INTEGRATION_CREDENTIALS" ]]; then echo "${INTEGRATION_CREDENTIALS}" > .integration-credentials.js; fi
```

integration-credentials should be in the extension directory (like in the bc-cart extension)
```suggestion
      - if [[ "$INTEGRATION_CREDENTIALS" ]]; then echo "${INTEGRATION_CREDENTIALS}" > extension/.integration-credentials.js; fi
```